main.cpp

```
@@ -12,6 +12,19 @@ using Catch::Matchers::Message;
 class Quadratic {
     double a, b, c;
 public:
+Quadratic ()
+{
+  a=1.0;
+  b=0.0;
+  c=0.0;
+  std::invalid_argument(){
+    if (a==0) throw invalid_argument
```

You need to throw an object of `invalid_argument` exception class.